README.md

```diff
@@ -1,801 +1,3854 @@
+      
+      <blockquote>
+          NOTE: The `kube_deploy_nodes.yaml` file is the configuration file used by odim-controller to set up a Kubernetes cluster and to deploy the Resource Aggregator for ODIM services. </blockquote>
+      
+      <blockquote>
+       Check the permissions of the archived tar files of the Docker images; the privilage of all the files must be `user:user`.
 
-6. Set up FQDN in the `/etc/hosts` file (only if there is no DNS infrastructure):
+## Building Docker images of all the services
 
-    a. Open the `/etc/hosts` file for editing:
+1. Run the following commands on the deployment node:
+   1. ```
+      $ git clone https://github.com/ODIM-Project/ODIM.git
       ```
-      $ sudo vim /etc/hosts
+      
+   2. ```
+      $ cd ODIM
       ```
-    b. Scroll to the end of the file, add the following line, and then save:
+      
+	3. ```
+      $ export ODIMRA_USER_ID=2021
       ```
-      <host_ipv4_address> <user_preferred_fqdn_for_host>
+	   
+   4. ```
+      $ export ODIMRA_GROUP_ID=2021
       ```
-   Example:
-`<host_ipv4_address> <fqdn>`
-
-7. Generate certificates:
+      
+   5. ```
+      $ ./build_images.sh
+	   ```
+	   
+	6. ```
+	   $ sudo docker images
+	   ```
+	   If the images are built successfully, you get an output which is similar to the following sample:
+	   
+	   | **REPOSITORY**  | **TAG** | **IMAGE ID** | **CREATED**  | **SIZE** |
+	   | --------------- | ------- | ------------ | ------------ | -------- |
+	   | consul          | 1.6     | 33ff2311df24 | 4 hours ago  | 185MB    |
+	   | odim_zookeeper  | 1.0     | 981d43f6c8b4 | 22 hours ago | 278MB    |
+	   | update          | 1.0     | 2cfb65430181 | 22 hours ago | 128MB    |
+	   | task            | 1.0     | c4dd52b9ade0 | 22 hours ago | 129MB    |
+	   | systems         | 1.0     | 9d3ad9845b16 | 22 hours ago | 129MB    |
+	   | redis           | 1.0     | 81bf552d3a52 | 2 days ago   | 99.2MB   |
+	   | managers        | 1.0     | 2f7955586c54 | 4 days ago   | 128MB    |
+	   | dellplugin      | 1.0     | 1601188cbd8f | 7 days ago   | 103MB    |
+	   | odim_kafka      | 1.0     | f03a52363483 | 8 days ago   | 278MB    |
+	   | grf-plugin      | 1.0     | c7a086d02b16 | 11 days ago  | 100MB    |
+	   | fabrics         | 1.0     | 9b9ea8bafc30 | 11 days ago  | 128MB    |
+	   | events          | 1.0     | 860a9202a483 | 11 days ago  | 130MB    |
+	   | aciplugin       | 1.0     | 0c42ba5d4223 | 6 weeks ago  | 114MB    |
+	   | urplugin        | 1.0     | fb3c1cf141d5 | 6 weeks ago  | 101MB    |
+	   | api             | 1.0     | effab530ede5 | 2 months ago | 130MB    |
+	   | aggregation     | 1.0     | 354f67a857b6 | 3 months ago | 130MB    |
+	   | account-session | 1.0     | a7eb07e69395 | 3 months ago | 129MB    |
+	
+2. Save each Docker image to a tar archive using the following command:
+    ```
+    $ sudo docker save -o <image_name.tar> <image_name:tag>
+    ```
+	Example: `sudo docker save -o consul.tar consul:1.0`
```

Comment:
consul tag is 1.6 so change example with sudo docker save -o consul.tar consul:1.6